content/news/buzz/2025-curtin-capstone-semester-end-blog/contents.lr

```diff
@@ -0,0 +1,109 @@
+# **DRAFT BLOG POST**
+
+title: 2025 Curtin Capstone Collaboration with BeeWare
+---
+author: Curtin Capstone Team
+---
+body:
+
+# Curtin Capstone Collaboration with Beeware 2025
+
+Over the past semester, our team of final year students from Curtin University has been collaborating with the Beeware open source project as part of our university’s Capstone Project.
+
+## Curtin Capstone
+
+Capstone is the final year project undertaken by students across all commuting disciplines, including Computer Science, Software Engineering, Cyber Security, and Information Technology. It allows students the opportunity to work in teams on real-world projects in collaboration with industry partners, gaining practical experience and professional exposure before graduating.
+
+This year our team has had the exciting opportunity to contribute to BeeWare for our Capstone project.
+
+Meet the Team:
+
+ - Kavidu Abeykoon Mudiyanselagedara
+	 - Discipline: Information Technology
+	 - GitHub: [kavi2du](https://github.com/kavi2du)
+ - Callum Horton
+	 - Discipline: Software Engineering
+	 - GitHub: [Stringer90](https://github.com/Stringer90)
+ - Caydn Lee
+	 - Discipline: Software Engineering
+	 - GitHub: [caydnn](https://github.com/caydnn)
+ - Jaeden Mah
+	 - Discipline: Computer Science
+	 - GitHub: [JMah007](https://github.com/JMah007)
+ - Mitchell Pontague
+	 - Discipline: Software Engineering
+	 - GitHub: [mEp3ii2](https://github.com/mEp3ii2)
+ - Veronica Taniputra
+	 - Discipline: Cyber Security
+	 - GitHub: [vt37](https://github.com/vt37)
+
+  ## What We’ve Worked On
```

Comment:
The markup goes all wonky here - it's indented in a weird way that doesn't render.